ch_tools/chadmin/cli/replica_group.py

```diff
@@ -278,47 +290,67 @@ def restore_command(
     else:
         replicas = list_table_replicas(ctx, is_readonly=True, **kwargs)
 
+    rotines: List[WorkerTask] = []
     for replica in replicas:
```

Comment:
`rotines` here and `tasks` in `data_store_group.py`. Why is it used different names for the same thing?

Reply:
Fixed